src/Microsoft.VisualBasic/src/Microsoft/VisualBasic/CompilerServices/ExceptionUtils.vb

```diff
@@ -2,14 +2,165 @@
 ' The .NET Foundation licenses this file to you under the MIT license.
 ' See the LICENSE file in the project root for more information.
 
+Option Strict Off
+
 Imports System
 Imports Microsoft.VisualBasic.CompilerServices.Utils
 
 Namespace Microsoft.VisualBasic.CompilerServices
 
     Friend Enum vbErrors
+        None = 0
```

Comment:
Obviously I didn't need all of these - but it seemed best to keep it all together.